docs/04-guides/on-demand/01-upload-video-asset/02-api.mdx

```diff
@@ -208,3 +210,101 @@ curl --location --request GET 'https://livepeer.studio/api/asset/{id}' \
 
 </CodeBlockTabItem>
 </CodeBlockTabs>
+
+## Resumable Upload
+
+The resumable upload method allows a user to resume their upload if their 
```

Comment:
Let's change the text describing resumable upload to the following:

"A resumable upload allows you to resume an upload operation after a communication failure interrupts the flow of data. Because you don't have to restart large file uploads from the start, resumable uploads can also reduce your bandwidth usage if there is a network failure.

Resumable uploads are useful when your file sizes might vary greatly or when there is a fixed time limit for requests. You might also use resumable uploads for situations where you want to show an upload progress bar."